<launch>
	<param name="/baud_rate" type="int" value="115200" />
	<param name="/qpps" type="string" value="0,0,0,0" />
	<param name="/accel" type="string" value="0,0,0,0" />
	<param name="battery_low" type="int" value="11" />
	<param name="battery_high" type="int" value="18" />
	<param name="mech_dist" type="string" value="7.254,10.5,10.5,10.073" />

	<node name="hardware_interface" pkg="rover_mini_control" type="hardware_interface.py" respawn="true" output="screen">
		<param name="encoder_min" value="-2147483648" />
		<param name="encoder_max" value="2147483647" />
		<param name="ticks_per_meter" value="26069" />
	</node>

	<node name="diff_drive" pkg="differential_drive" type="diff_drive.py" output="screen">
                <remap from="cmd_vel" to="mux_cmd_vel"/>
                <param name="publish_tf" value="false"/>
		<param name="base_width" value="0.425" />
		<param name="ticks_per_meter" value="26069" />
		<param name="encoder_min" value="-2147483648" />
		<param name="encoder_max" value="2147483647" />
	</node>
</launch>
